config/environments/development.rb

```diff
@@ -1,18 +1,25 @@
+require "active_support/core_ext/integer/time"
```

Comment:
I've regenerated all env files using Rails 7 defaults and then backported all changes from old files

This will simplify future upgrades